pkg/function/backup_data_stats.go

```diff
@@ -191,6 +191,18 @@ func (b *BackupDataStatsFunc) Exec(ctx context.Context, tp param.TemplateParams,
 		return nil, err
 	}
 
+	if tp.PodAnnotations != nil {
+		// merge the actionset annotations with blueprint annotations
+		var actionSetAnn ActionSetAnnotations = tp.PodAnnotations
+		annotations = actionSetAnn.MergeBPAnnotations(annotations)
```

Comment:
```suggestion
		ActionSetAnnotations(tp.PodAnnotations).MergeBPAnnotations(annotations)
```
Can we do something like this? Since map is passed by ref, the map will be updated in the function. We don't have to return value.